docs/content/operations/api-reference.md

```diff
@@ -498,7 +498,49 @@ This section documents the API endpoints for the processes that reside on Data s
 
 ### MiddleManager
 
-The MiddleManager does not have any API endpoints beyond the [common endpoints](#common).
+##### GET
+
+* `/druid/worker/v1/enabled`
+
+Check whether a MiddleManager is in an enabled or disabled state. Returns JSON object keyed by the `druid.host` and the boolean state.
+
+```json
+{"localhost":true}
```

Comment:
I'm sure it returns `host:port`.

Reply:
Oops, yes you are correct, thanks for catching! 👍